tests/TestCase.php

```diff
@@ -36,6 +36,7 @@ public function setUp()
 
         // Setup mocks
         $this->fileSystem = $this->mock('Illuminate\Filesystem\Filesystem');
+        $this->phoenix = $this->mock('Rogue\Services\Phoenix\Phoenix');
```

Comment:
Two very nit-picky suggestions:

1. Let's use PHP's `::class` here so it's easy to see whether these resolve or not, like this `$this->mock(\Rogue\Services\Phoenix\Phoenix::class);`
2. The Phoenix class looks like it's "extra" namespaced! Let's remove the [extraneous `\Phoenix` namespace](https://github.com/DoSomething/rogue/blob/0f92aed400f28c9d79a394e1289bbb00604467ad/app/Services/Phoenix/Phoenix.php#L3) from the class so we don't need to type that twice.